pulp_puppet_plugins/pulp_puppet/plugins/db/models.py

```diff
@@ -176,7 +168,7 @@ def split_filename(filename):
                 # This is the forge format, but Puppet still allows it
                 author, name = filename.split("/", 1)
             except ValueError:
-                raise InvalidModuleName(filename)
+                raise PulpCodedException(error_code=error_codes.PUP0004, name=filename)
```

Comment:
I'm not sure if this should become a class defined exception or left as-is. Just a thought.